package.json

```diff
@@ -120,5 +120,8 @@
         "babel-module": {}
       }
     }
+  },
+  "dependencies": {
+    "normalize-url": "^1.9.0"
```

Comment:
Let's not introduce unnecessary dependencies please.

You can simply fix this "problem" with a `replace(/\/$/, '')`

```js
'foo/'.replace(/\/$/, '')
// "foo"
'foo'.replace(/\/$/, '')
// "foo"
```